dev-docs/bidders/pubmatic.md

```diff
@@ -62,6 +62,30 @@ pbjs.setConfig({
 
 Note: Combine the above the configuration with any other UserSync configuration. Multiple setConfig() calls overwrite each other and only last call for a given attribute will take effect.
 
+### adSlot Specification and Multi-Size Ad Units
+
+The adSlot parameter supports two different formats:
+
+{: .table .table-bordered .table-striped }
+| Format         | Example              |
+|----------------|----------------------|
+| Without Size   | `'38519891'`         |
+| With Size      | `'38519891@300x205'` |
+
+Support and behavior differs by Prebid.js version
+
+Prebid.js v2.9.0 and later:
+
+Both formats are supported.  Without Size is the recommended unless you are using Prebid Server, in which case, With Size is required.  Both formats will send the ad request with all sizes specified in the Prebid ad unit configuration.
+
+Prebid.js v1.1.18 to v2.0.8:
```

Comment:
I think you mean 2.8.0?

Reply:
Yes, I did mean 2.8.0.  Thanks for flagging this!